aiozmq/stream.py

```diff
@@ -123,12 +126,15 @@ def __init__(self, loop, *, high=None, low=None):
         self._protocol = ZmqStreamProtocol(self, loop=loop)
         self._loop = loop
         self._queue = collections.deque()
+        self._event_queue = collections.deque()
```

Comment:
Please add constructor parameter for managing `_event_queue` size. 
I guess default should be big enough -- say, 100 events (which is not too big for taking care about memory consumption). `None` is not allowed -- I'd like to reserve `None` for adding flow control later if needed.